### ISSUE TITLE
Should the editor toolbar have a role=toolbar?

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#632]
More something to discuss with accessibility people and anyone interested, than a real technical issue. Speaking about these toolbars:

<img width="619" alt="toolbar" src="https://cloud.githubusercontent.com/assets/1682452/25672612/a04a9064-3034-11e7-9f25-132113493515.png">

#631 aims to provide arrow navigation, so should they also have an appropriate ARIA role?

**Comment by afercia** [comment]
This applies also to other groups of controls, for example the text editor quicktags buttons. They're a typical example of "toolbar" ad described in the WAI-ARIA Authoring Practices 
https://www.w3.org/TR/wai-aria-practices/#toolbar

ideally they should be grouped in an element with role="toolbar" and implement arrow navigation (see link above)

**Comment by afercia** [comment]
Many things have changed: there is now just one "dynamic" fixed toolbar at the top. Further improvements and discussion on interaction and ARIA roles are now being discussed on https://github.com/WordPress/gutenberg/issues/3383
